libr/util/format.c

```diff
@@ -506,7 +508,11 @@ static void r_print_format_time(const RPrint* p, int endian, int mode,
 	if (MUSTSET) {
 		p->cb_printf ("wv4 %s @ 0x%08"PFMT64x"\n", setval, seeki+((elem>=0)?elem*4:0));
 	} else if (MUSTSEE) {
-		char *timestr = strdup(asctime (gmtime_r ((time_t*)&addr, &timestruct)));
+		char *timestr = malloc (ASCTIME_BUF_MINLEN);
+		if (!timestr) {
+			return;
+		}
+		asctime_r (gmtime_r ((time_t*)&addr, &timestruct), timestr);
```

Comment:
dont use asctime_r. its not portable. use the old implementation when on __WINDOWS__

this will fix the CI

Reply:
oh wait. as long as this same code is called several times it will be better to exposae this API in util/time.c and use it in every call instead of having to ifdef for windows all the time

Reply:
@trufae Okay, I'll do it tomorrow. Sorry for the wait.